sdk/eventhub/azure-eventhub/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 5.10.1 (Unreleased)
+
+### Bugs Fixed
+
+### Other Changes
+
+- Internal refactoring to support upcoming Pure Python AMQP-based release.
```

Comment:
not sure if we should say this at all.

Reply:
I think its fair game since we plan on promoting the alpha in the changelog. Changes for something coming in the future, that is known is ok